doc_utils/read_utils.py

```diff
@@ -1,12 +1,28 @@
 import pandas as pd
-from typing import Dict, List, Any
-from .errors import MissingFieldError
+
+from typing import Any, Dict, FrozenSet, List, Optional, Set, TypeVar
+
+from .errors import InvalidTreatment, MissingFieldError
+from .typing import T
 
 class DocReadUtils:
     """This is created as a Mixin for others to easily add to their classes
     """
+    missing_treatments = frozenset((
+        "return_empty_string",  # return ""
+        "return_none",          # return None
+        "replace",              # replace missing value with specified value
+        "raise_error",          # raises MissingFieldError
+    ))
+
     @classmethod
-    def get_field(self, field: str, doc: Dict, missing_treatment: bool='raise_error'):
+    def get_field(
+        cls, 
+        field: str, 
+        doc: Dict, 
+        missing_treatment: str='raise_error',
```

Comment:
given that we defined a missing treatment type - shouldn't this be updated as well?